.github/ISSUE_TEMPLATE/v2-bug-report.md

```diff
@@ -32,9 +32,9 @@ A clear and concise description of what the bug is.
 
 Describe the steps or code required to reproduce the behavior
 
-## Expected behavior
```

Comment:
Based on the `v1` template, I'm guessing we want both "Expected" and "Observed" here as well. Looks like "Expected" is being deleted.

Reply:
Ah, this was a copy paste issue on my part. Will fix, nice catch 🦅